src/leveled_cdb.erl

```diff
@@ -489,48 +470,56 @@ starting({open_writer, Filename}, _From, State) ->
                             last_key=LastKey,
                             filename=Filename,
                             hashtree=HashTree},
-    {reply, ok, writer, State0, hibernate};
-starting({open_reader, Filename}, _From, State) ->
+    {next_state, writer, State0,
+     [{reply, From, ok}, hibernate]};
+starting({call, From}, {open_reader, Filename}, State) ->
     leveled_log:save(State#state.log_options),
     leveled_log:log(cdb02, [Filename]),
     {Handle, Index, LastKey} = open_for_readonly(Filename, false),
     State0 = State#state{handle=Handle,
                             last_key=LastKey,
                             filename=Filename,
                             hash_index=Index},
-    {reply, ok, reader, State0, hibernate};
-starting({open_reader, Filename, LastKey}, _From, State) ->
+    {next_state, reader, State0,
+     [{reply, From, ok}, hibernate]};
+starting({call, From}, {open_reader, Filename, LastKey}, State) ->
     leveled_log:save(State#state.log_options),
     leveled_log:log(cdb02, [Filename]),
     {Handle, Index, LastKey} = open_for_readonly(Filename, LastKey),
     State0 = State#state{handle=Handle,
                             last_key=LastKey,
                             filename=Filename,
                             hash_index=Index},
-    {reply, ok, reader, State0, hibernate}.
-
-writer({get_kv, Key}, _From, State) ->
-    {reply,
+    {next_state, reader, State0,
+     [{reply, From, ok}, hibernate]};
+starting({call, From}, Event, State) ->
+    handle_sync_event(Event, From, State);
+starting(cast, clerk_complete, _State) ->
+    {keep_state_and_data, [hibernate]};
+starting(info, _Msg, _State) ->
+    {keep_state_and_data, []}.
+
```

Comment:
If there are no `info` messages supported should we remove handling of them.  Again code coverage being the issue.